client/src/leap/soledad/client/encdecpool.py

```diff
@@ -507,15 +446,12 @@ def insert_encrypted_received_doc(
         :param idx: The index of this document in the current sync process.
         :type idx: int
```

Comment:
Updated and merged with async_decrypt_doc. Btw, it still inserts.
Staging on decryption had two inserts, one for encrypted and another for decrypted.
I just jumped directly to the decrypted insert, but it is needed to sort out the sequence as before (but now in memory :))